docker/README.md

```diff
@@ -37,18 +48,27 @@ containers to communicate with each other - it prevents worker to connect to Web
 
 on host machine.
 
-## Create directory structure
+**Create directory structure**
+
+In case you want to have the big files (isos and disk images, test and needles) outside of the docker volume container,
+you should create this file structrure from within the directory you are going to execute the container.
 
     mkdir -p data/factory/{iso,hdd} data/tests
 
-It is also necessary to either run all containers in privileged mode, or set selinux properly:
+It could be necessary to either run all containers in privileged mode, or set selinux properly. If you are having problems with it, run this command:
```

Comment:
Actually the question is under which circumstances is it necessary. I have never needed to execute this command to make it work. And I only need to execute the worker container in privileged mode, so the virtual machine inside the container can work properly. I would remove this paragraph, but I preferred to maintain this information as hint in case someone eventually get this error. So I changed the "must do" speech to "do it if you get the error" speech.